src/App.tsx

```diff
@@ -535,6 +550,82 @@ class Map {
       }
     }
   }
+
+  getCastlePoints(firstX: number, firstY: number, octaquadrant: number)  {
+    let thisPiece = this.getAt(firstX, firstY).piece
+    console.log(thisPiece.sideConnections[octaquadrant])
+
+    /* if (thisPiece.sideTypes[Math.floor((octaquadrant)/2)] === 2) {
+          return
+        } */
+
+      interface Quadrant {
```

Comment:
This interface will be used elsewhere. Move it to the top level like all the other interfaces